pkg/rook/common/logs.go

```diff
@@ -8,11 +8,20 @@ import (
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
 )
 
-var ValidCephSubSystems = []string{"osd", "mds", "mon", "mgr", "auth"}
+var (
+	ValidCephSubSystems = []string{"default", "lockdep", "context", "crush", "mds", "mds_balancer", "mds_log", "mds_log_expire",
```

Comment:
Let's not validate the subsystems. We don't want to maintain this list which will change over time.

Reply:
What if an invalid subsystem is passed to ceph? Will the ceph command fail or ignore the bad setting?

Reply:
ceph will fail if invalid subsystem is passed

```
❯ bin/odf set log-level osd.0 invalid 30
Error: command terminated with exit code 22
Error EINVAL: unrecognized config option 'debug_invalid'

```